lib/explorer/backend/lazy_series.ex

```diff
@@ -524,6 +525,17 @@ defmodule Explorer.Backend.LazySeries do
     end
   end
 
+  @impl true
+  def cumulative_count(%Series{} = series, reverse) do
+    args = [lazy_series!(series), reverse]
+
+    if aggregations?(args), do: raise_agg_inside_window(:cumulative_count)
+
+    data = new(:cumulative_count, args, {:u, 32}, false)
+
+    Backend.Series.new(data, {:u, 32})
```

Comment:
The `cum_sum` value seems to return a u32, which was causing some mischief when using the same dtype as the underlying series. So here we define the function explicitly to control the dtype, rather than creating it by looping through the module attribute that has the other window functions.